src/content/docs/style-guide/writer-workflow/github-intro.mdx

```diff
@@ -0,0 +1,154 @@
+* Assignee: The person taking responsibility for a PR or issue. This will usually be used by the Hero or Sidekick to assign non-TW PRs and issues to themselves. It can also be used to take a TW’s PR or issue over from them.
+* Reviewer: The person who reviews or peer edits the code/document and approves the changes. Not necessarily the person responsible for that area or responsible for merging the commit. You can pre-assign up to 100 reviewers to a given issue.
+
+## Track issues in the board [#github-board] 
+The docs board has the following columns:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "300px" }}>
+        Column
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        **Needs triage**
+      </td>
+      <td>
+        The Hero or Sidekick review issues and PRs in this column, then drag them to the appropriate column.
+
+        If the issue/PR is labeled `eng`, go ahead and click its ellipses icon to archive it. For other issues and PRs, add these labels as applicable:
+        * `content`
+        * `internal`, when issued by a New Relic employee
+        * `external`, when issued by someone outside New Relic, or you're not sure they are an employee
+        * `Jira’d`, if you create a Jira from the issue  
+
+        If you create a Jira, don't close the original issue/PR, but feel free to remove it from the board until you work on it again.
+        
+        If you open an PR based on an external issue, label that PR as internal since you're doing the work.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Hero: to do**
+      </td>
+      <td>
+        PRs that the Hero needs to review, publish, and  follow up with SMEs as needed.  
+        
+        **Assign yourself as Assignee**.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **In review (Hero or any TW)**
+      </td>
+      <td>
+        Drag PRs to this column when they are being reviewed. This shows who is reviewing and what is being reviewed, so two writers don’t mistakenly work on the same PR.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Any TW: Writer needs PR review**
+      </td>
+      <td>
+        PRs from Tech Docs team members that need a light edit pass to make sure everything in GitHub is correct. This should be checked by other writers every few hours so PRs don’t get stale. 
+        
+        If you have a PR that’s been lingering here too long, ask for a reviewer in #doc_sprint_talk. 
+        
+        **Whoever takes it: assign yourself as Reviewer.**
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Any TW: needs peer edit**
+      </td>
+      <td>
+        Like our Needs Peer Edit column in Jira: A writer has requested a review of their PR. Review their PR in GitHub and leave comments. 
+        
+        **Whoever takes it: assign yourself as Reviewer.**
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Waiting on SME**
```

Comment:
Should rename it to `Waiting on SME/Blocked`?

Reply:
Yes!